### DISCUSSION TITLE
Procedures without the model name

### DISCUSSION THREAD

**Discussion by emab** [omar-dulaimi/prisma-trpc-generator#63] [Ideas]
I've implemented this in a fork here:

- https://github.com/omar-dulaimi/prisma-trpc-generator/compare/master...emab:prisma-trpc-generator:master

The reason I feel this is a useful config feature is that it gives you a more predictable set of procedures for a given router.

Since routers already include the model name I also feel it can be overkill to use the model name multiple times:

```ts
api.accountsRouter.createOne({ ... });

// vs

api.accountsRouter.createOneAccount({ ... });
```

Interested to know your thoughts!

**Comment by maxime4000** [comment]
I think this would be great to add. Please make a PR about it :)

**Comment by omar-dulaimi** [comment]
This sounds great
